### DISCUSSION TITLE
dns do github,io

### DISCUSSION THREAD

**Discussion by douglastos** [DaniloCarSan/danilocarsan.github.io#11] [General]
Ola Danilo segui seu howto https://danilocarsan.github.io/Posts/Github/Gitub-Pages/Como-criar-um-blog-usando-Github-Pages-+-Obsidian-+-Quartz porem não encontrei onde eu deixo somente a url sem o nome do projeto ex:
https://douglastos.github.io/douglas.github.io/

consegue me explicar isso?

**Comment by DaniloCarSan** [comment]
Opa tudo bem, para que funcione corretamente você precisa dar ao seu repositório o mesmo nome de seu usuário:

Atual:  douglas.github.io

Como deve ser: douglastos.github.io

Teste e vê se funciona

**Reply by douglastos** [reply]
Boa era isso mesmo... nao me atentei a isso, obrigado.
